src/index.html

```diff
@@ -10,23 +10,34 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Airbnb Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body>
+    <form
+      class="search-bar"
+      data-qa="big"
+    >
+      <input
+        class="search-bar__input"
+        data-qa="keypress"
+        type="text"
+        placeholder="Try “Los Angeles”"
```

Comment:
Use semantic tags where possible. Consider wrapping the `<input>` elements in a semantic tag like `<section>` or `<div>` to indicate a block of related content.